data/saved/transactions.csv

```diff
@@ -0,0 +1,101 @@
+2018-09-20,000aa7f0dc06cd7174389e76c9e132a67860c5f65f970699daccc14425ac31a8,553139001,0.0338813559322033,2
+2018-09-20,000aa7f0dc06cd7174389e76c9e132a67860c5f65f970699daccc14425ac31a8,377277001,0.008457627118644,2
+2018-09-20,000aa7f0dc06cd7174389e76c9e132a67860c5f65f970699daccc14425ac31a8,377277001,0.008457627118644,2
+2018-09-20,000aa7f0dc06cd7174389e76c9e132a67860c5f65f970699daccc14425ac31a8,700819006,0.0423559322033898,2
+2018-09-20,000aa7f0dc06cd7174389e76c9e132a67860c5f65f970699daccc14425ac31a8,700819006,0.0423559322033898,2
+2018-09-20,001127bffdda108579e6cb16080440e89bf1250a776c6e55f56e35e9ee029a8d,397068015,0.0338813559322033,1
+2018-09-20,001ea4e9c54f7e9c88811260d954edc059d596147e1cf8adc73323aebf571fd8,652075001,0.0118474576271186,2
+2018-09-20,001ea4e9c54f7e9c88811260d954edc059d596147e1cf8adc73323aebf571fd8,670295001,0.0101525423728813,2
+2018-09-20,001fd23db1109a94bba1319bb73df0b479059027c182da490e1161b34cd3af61,631744002,0.0169322033898305,1
+2018-09-20,001fd23db1109a94bba1319bb73df0b479059027c182da490e1161b34cd3af61,562252035,0.021593220338983,1
+2018-09-20,0021da829b898f82269fc51feded4eac2129058ee95bd75bb1591e2eb14ecc79,649356002,0.0274406779661017,1
+2018-09-20,0021da829b898f82269fc51feded4eac2129058ee95bd75bb1591e2eb14ecc79,579941002,0.0198135593220338,1
+2018-09-20,0021da829b898f82269fc51feded4eac2129058ee95bd75bb1591e2eb14ecc79,629760002,0.0152372881355932,1
+2018-09-20,0021da829b898f82269fc51feded4eac2129058ee95bd75bb1591e2eb14ecc79,625229004,0.0198135593220338,1
+2018-09-20,00228762ecff5b8d1ea6a2e52b96dafa198febddbc3bf350eb611f28e61ea6ce,688545001,0.1342372881355932,2
+2018-09-20,002b3c0a44a22c45a8d62ea9d2b88d1a89e335f8b8400335a85c3b5845794cb9,673531001,0.008457627118644,2
+2018-09-20,002b3c0a44a22c45a8d62ea9d2b88d1a89e335f8b8400335a85c3b5845794cb9,464277014,0.0220169491525423,2
+2018-09-20,002b3c0a44a22c45a8d62ea9d2b88d1a89e335f8b8400335a85c3b5845794cb9,464277014,0.0220169491525423,2
+2018-09-20,003ac6eff3ca20cbe4e14a106a5e28b7551616005de8f9dd0f9455e5543b31ac,617322003,0.0338813559322033,1
+2018-09-20,003ac6eff3ca20cbe4e14a106a5e28b7551616005de8f9dd0f9455e5543b31ac,617322003,0.0186271186440677,1
+2018-09-20,003ac6eff3ca20cbe4e14a106a5e28b7551616005de8f9dd0f9455e5543b31ac,620405001,0.0186271186440677,1
+2018-09-20,003ac6eff3ca20cbe4e14a106a5e28b7551616005de8f9dd0f9455e5543b31ac,678239001,0.0254067796610169,1
+2018-09-20,00401a367c5ac085cb9d4b77c56f3edcabf25153615db982fcc7991d182e10a9,613456009,0.0169322033898305,2
+2018-09-20,00401a367c5ac085cb9d4b77c56f3edcabf25153615db982fcc7991d182e10a9,633675001,0.0101525423728813,2
+2018-09-20,00401a367c5ac085cb9d4b77c56f3edcabf25153615db982fcc7991d182e10a9,648719001,0.0254067796610169,2
+2018-09-20,00401a367c5ac085cb9d4b77c56f3edcabf25153615db982fcc7991d182e10a9,427114015,0.0169322033898305,2
+2018-09-20,00401a367c5ac085cb9d4b77c56f3edcabf25153615db982fcc7991d182e10a9,567475001,0.0338813559322033,2
```

Comment:
I don't think we should commit these csvs, no?

Reply:
Yes, deleted.